doc/ubuntu_release_process.md

```diff
@@ -156,6 +156,12 @@ build-now.
 This is generally needed when we are disabling backported feature from tip
 in order to retain existing behavior on an older series.
 
+**Note:** If adding a quilt patch to modify functionality introduced by
+a debian/patches/cpick-\* file, the new quilt patch needs the prefix
+fix-cpick-<previous_cpick_filename> because new-upstream-snapshot removes all
+cpick-\* and fix-cpick-\* files from debian/patches directory. Those patches
+are already considered "applied" in the snapshot we are syncing.
```

Comment:
'considered "applied"' is a weaker statement than we can confidently make here: we know those patches were cherry-picked from commits that are now being snapshot, so we know that the patches are no longer necessary.

Reply:
Changed to 
```suggestion
are already applied in the snapshot we are syncing.
```